app/bundles/LeadBundle/Tests/EventListener/PageSubscriberTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\LeadBundle\Tests\EventListener;
+
+use Mautic\CoreBundle\Tests\CommonMocks;
+use Mautic\LeadBundle\Entity\Lead;
+use Mautic\LeadBundle\EventListener\PageSubscriber;
+use Mautic\PageBundle\Entity\Page;
+use Mautic\PageBundle\Event\PageDisplayEvent;
+use Mautic\PageBundle\PageEvents;
+
+class PageSubscriberTest extends CommonMocks
+{
+    public function testSubscribedEvents(): void
+    {
+        $this->assertSame(
+            [
+                PageEvents::PAGE_ON_DISPLAY => ['replaceContactFieldTokenOnPageDisplay', 0],
+            ],
+            PageSubscriber::getSubscribedEvents()
+        );
+    }
+
+    public function replaceContactFieldTokenOnPageDisplay(): void
```

Comment:
```suggestion
    public function testReplaceContactFieldTokenOnPageDisplay(): void
```

Tests must have the `test` prefix in order to run properly.